src/Umbraco.Tests/Services/UserServiceTests.cs

```diff
@@ -452,6 +452,24 @@ public void Can_Remove_Section_From_All_Assigned_Users()
         }
 
         [Test]
+        public void Can_Add_Section_To_All_Users()
+        {
+            var userType = ServiceContext.UserService.GetUserTypeByAlias("admin");
+
+            var user1 = ServiceContext.UserService.CreateUserWithIdentity("test1", "test1@test.com", userType);
+            var user2 = ServiceContext.UserService.CreateUserWithIdentity("test2", "test2@test.com", userType);
+
+            //now add the section to all users
+            ServiceContext.UserService.AddSectionToAllUsers("test");
+
+            //assert
+            var result1 = ServiceContext.UserService.GetUserById((int)user1.Id);
+            var result2 = ServiceContext.UserService.GetUserById((int)user2.Id);
+            Assert.IsFalse(result1.AllowedSections.Contains("test"));

```

Comment:
If you are adding the "test" section to all users shouldn't this be `Assert.IsTrue` ?